### PR TITLE
Add support for multiple deployment and use proxy

### PR DESCRIPTION
**Pull request by mabuaisha**

### CODE REVIEW

**Comment by EarthmanT** [comment]
Blueprints should now use the 4.2+ plugin import notation, e.g.:

```
  - plugin:cloudify-diamond-plugin
  - plugin:cloudify-fabric-plugin
  - plugin:cloudify-utilities-plugin
  - plugin:cloudify-kubernetes-plugin
```

Don't specify plugin version, and don't support full URL.

**Comment by EarthmanT** [comment]
What changes need to be made to the [simple blueprint](https://github.com/cloudify-examples/simple-kubernetes-blueprint).

**Comment by EarthmanT** [comment]
Expose Service fails:
```
[centos@server-test001-k8s-master-host-ji4nla ~]$ kubectl describe svc/wordpress
Name:                     wordpress
Namespace:                default
Labels:                   app=wordpress
Annotations:              <none>
Selector:                 app=wordpress,tier=frontend
Type:                     LoadBalancer
IP:                       10.97.102.205
Port:                     <unset>  80/TCP
TargetPort:               80/TCP
NodePort:                 <unset>  30080/TCP
Endpoints:                10.44.0.5:80
Session Affinity:         None
External Traffic Policy:  Cluster
Events:
  Type     Reason                      Age                From                Message
  ----     ------                      ----               ----                -------
  Normal   EnsuringLoadBalancer        4m (x10 over 24m)  service-controller  Ensuring load balancer
  Warning  CreatingLoadBalancerFailed  4m (x10 over 24m)  service-controller  Error creating load balancer (will retry): failed to ensure load balancer for service default/wordpress: no groups for scale
```

File: .gitignore [hunk 1]
```diff
@@ -1,22 +1,105 @@
...
 
-# python
-*.pyc
+# mypy
+.mypy_cache/
 
-# Ignore cfy-kubernetes cluster-autoscaler
-examples/cluster_blueprint/resources/cfy-kubernetes
-examples/cluster_blueprint/resources/cfy-autoscale
-examples/cluster_blueprint/resources/cfy-go
```
Comment (0lvin) [inline .gitignore]: We should ignore all binary files.

**Comment by 0lvin** [comment]
@mabuaisha please resolve conflicts.

**Comment by mabuaisha** [comment]
@0lvin the conflicts resolved please take a look to the iaas blueprints imports to make sure we are on the same page for imports

File: .gitignore (hunk 1 above)
Comment (mabuaisha) [reply .gitignore]: Done

**Pull request merged**